deposit/src/test/java/edu/unc/lib/deposit/fcrepo4/IngestContentObjectsJobIT.java

```diff
@@ -211,7 +217,27 @@ public void ingestWorkObjectTest() throws Exception {
 
 		// Verify the properties and content of the supplemental file
 		assertBinaryProperties(supObj, FILE2_LOC, FILE2_MIMETYPE, null, FILE2_SIZE);
-
+		// Verify that ingestion event gets added for work
+		List<PremisEventObject> eventsWork = mWork.getPremisLog().getEvents();
+		assertEquals(2, eventsWork.size());
+		
+		assertTrue(eventsWork.contains(findPremisEventByEventDetail(eventsWork, "ingested as PID: " + mWork.getPid())));
+		assertTrue(eventsWork.contains(findPremisEventByEventDetail(eventsWork, "added 2 child objects to this container")));
+		
+		// Verify that ingestion event gets added for primary object
+		List<PremisEventObject> eventsPrimObj = primaryObj.getPremisLog().getEvents();
+		assertEquals(1, eventsPrimObj.size());
+		assertEquals("ingested as PID: " + primaryObj.getPid().toString() + "\n ingested as filename: "
```

Comment:
this is a meaty assertion :)  I believe you could cut down a bit on the length by using mainPid and FILE1_LOC instead of making requests against the primaryObj since that's not what's being tested here.  I wonder if it might be a bit more readable to use your awesome lambda helper method too?